lib/components/typography/src/body.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import c from "classnames";
+
+// @ts-ignore
+import s from "./styles.module.less";
+import { BaseProps } from ".";
+
+export interface Props extends BaseProps {
+  fontSize?: "14" | "16";
```

Comment:
is simply a small flag better here? Or t-shirt sizes? Went with 14 | 16 as it's the language we use in the bo system